src/doc/rustc/src/platform-support/loongarch-linux.md

```diff
@@ -0,0 +1,87 @@
+
+|    `<fabi-suffix>`     |                           `Description`                            |
+|------------------------|--------------------------------------------------------------------|
+|          f64           | The base ABI use 64-bits FPRs for parameter passing.(lp64d, ilp32d)|
+|          f32           | The base ABI uses 32-bit FPRs for parameter passing. (lp64f ilp32f)|
+|          sf            | The base ABI uses no FPR for parameter passing. (lp64s ilp32s)     |
+
+|   `<abiext-suffix>`    |                       `ABI extension type`                         |
+|------------------------|--------------------------------------------------------------------|
+|    (empty string)      |                               base                                 |
+
+|`ABI type(Base ABI/ABI extension)`| `C library` | `kernel` |          `target tuple`          |
+|----------------------------------|-------------|----------|----------------------------------|
+|           lp64d/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf64 |
+|           lp64f/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf32 |
+|           lp64s/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnusf  |
+|           ilp32d/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf64 |
+|           ilp32f/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf32 |
+|           ilp32s/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnusf  |
+|           lp64d/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf64|
+|           lp64f/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf32|
+|           lp64s/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslsf |
+|           ilp32d/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf64|
+|           ilp32f/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf32|
+|           ilp32s/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslsf |
+
+## Target maintainers
+
+- zhaixiaojuan, `zhaixiaojuan@loongson.cn`, https://github.com/zhaixiaojuan
+
+## Requirements
+
+This target is cross-compiled.
+A platform-provided C compiler toolchain is required, and obtaining address of the cross toolchain [GNU LoongArch toolchain](https://github.com/loongson/build-tools/releases).
+
+## Building the target
+
+The target can be built by enabling it for a `rustc` build.
+
+```toml
+[build]
+target = ["loongarch64-unknown-linux-gnuf64"]
+```
+
+Make sure `loongarch64-unknown-linux-gnu-gcc` is included in `$PATH`. Alternatively, you can use GNU LoongArch Toolchain by adding the following to `config.toml`:
+
+```toml
+[target.loongarch64-unknown-linux-gnuf64]
+cc = "loongarch64-unknown-linux-gnu-gcc"
```

Comment:
```suggestion
cc = "/path/to/loongarch64-unknown-linux-gnu-gcc"
```

I think it's better to use an absolute path for these examples?  With no explicit path it's same as a search in `$PATH`.